sound/soc/intel/boards/sof_rt5682.c

```diff
@@ -269,6 +274,10 @@ static int sof_rt5682_hw_params(struct snd_pcm_substream *substream,
 			clk_freq = 24000000;
 		else
 			clk_freq = 19200000;
+
```

Comment:
Is there any chance that mclk is not configured in the topology? If not, we can remove `clk_freq =` above.

Reply:
> Is there any chance that mclk is not configured in the topology? If not, we can remove `clk_freq =` above.

I am not quite sure about that. I guess yes we can remove that, but that looks more invasive and might require testing on all related platforms.

Reply:
mclk is set by topology, so if topology defines a non zero mclk we should use it.